TestModels/Constraints/runtimes/java/build.gradle.kts

```diff
@@ -65,3 +68,7 @@ tasks {
         classpath = sourceSets["test"].runtimeClasspath
     }
 }
+
+tasks.named<Test>("test") {
+    useJUnitPlatform()
+}
```

Comment:
Ah interesting I wasn't aware. I only went with JUnit as a fairly arbitrary default, so I'll just switch to TestNG to avoid unnecessary complexity.

Reply:
If the Scope of Smoke Test does not include "consuming" libraries, this issue can be resolved.

Reply:
The primary focus is definitely testing smithy-dafny itself since that's where it's the most valuable, and given it can't support @references I don't see it being useful for a lot of library operations. But it's very easy to switch to TestNG to avoid the potential incompatibility down the road so I'm happy to do it.